charters/DataOperations/charter.md

```diff
@@ -0,0 +1,92 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions
+
+Data contribution: a set of raw data derived from the same experimental process, and the metadata describing that process
+Data release: the point at which metadata and data from a single data contribution become publicly available
+Data distribution: the packaging of metadata and data from multiple data contributions in meaningful ways and any announcements or documentation supporting it
+
+## Objectives
+
+The objective of the Data Operations is to provide the scientific community with effective tools and resources from the DCP which maximize the value and use of HCA data
+
+## In-scope
+
+- Data operations
+  - Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data
+  - Coordinate DCP-wide end-to-end testing pre-release and continued QC checks post-release to ensure data and data resource integrity and useability through component updates
+  - Identify data release- and data distribution-associated engineering needs of all DCP components and participate in crafting technical specifications
+  - Establish and integrate into the DCP roadmap a reproducible plan that includes the specifications for and the timing of iterative data distribution to the community
+  - Monitor the status of data contributions at each step in the DCP data workflow from initial identification of suitability through data release and potential inclusion in additional data distribution
```

Comment:
What is meant by identification of suitability? I don't think recruiting new data into the system is necessarily explicitly called out in the ingest support charter but that early pre-submitted to the DCP tracking of things is something I thought we agreed was a wrangler role rather than a data operations team role. Very happy to talk about what how to best share this responsibility (as I think it will be best shared) but it might be good to explicitly call it out and update the ingest support charter to do the same

Reply:
I think _monitor_ is the operative word here. We won't be contacting labs, determining suitability, or identifying what data to bring in (as noted below), but we will be aware of the processes involved in such decisions and aware of the current state. This will be similar to when data are in other steps of the workflow - we aren't responsible for pipeline running, but we will understand the 2ry analysis process, track the data while in 2ry analysis, etc.

Reply:
Okay, I think there is probably a better way to phrase this then as the term "initial identification of suitability" is easy to read as meaning you are involved in that process

Maybe something like "Monitor the status of data contribution from initial project notification through data release and potential inclusion in additional data distrubtions"

There might be a word for notification, I say this because we will probably track projects that are ultimately deemed unsuitable (or not something we can take yet)